vehicle/src/main/scala/com/wedriveu/vehicle/entity/SelfDrivingVehicle.scala

```diff
@@ -0,0 +1,45 @@
+package com.wedriveu.vehicle.entity
+
+
+import com.wedriveu.services.shared.utilities.Position
+import com.wedriveu.vehicle.control.VehicleControl
+
+/**
+  * Created by Michele on 28/07/2017.
+  */
+class SelfDrivingVehicle(plate: String, stateGiven: String, positionGiven: Position, batteryGiven: Double) {
```

Comment:
Please, check if this class can be implemented in less code lines, by maybe using a case-class